CHANGELOG.md

```diff
@@ -1,77 +1,77 @@
+- Fixed a bug with shift+scroll on small menus which would allow moving off the end of the menu, causing crashes.
+- Fixed several bugs with pad grid rendering.
+
+In addition, a number of improvements have been made to how the OLED display is used:
+
+- Added parameter name to Mod (Gold) Encoder popups.
+- "ARRANGER VIEW" and "SONG VIEW" now display the name of the current view on the screen.
+- The 12TET note name is now displayed along with the MIDI note number.
+- Added a new community setting which allows emulating the old 7SEG style on the OLED display.
+- Fixed several cases where popups could get stuck open.
+- Fixed a number of minor rendering bugs.
+
+### Sequencer
+
+- When changing instrument presets in session or arranger view all presets are now changed. Individual clips can still have their presets changed by first entering the clip.
+- Added support for 5 and 6 note scales.
+- Added 8 new built-in scales: Melodic Minor, Hungarian Minor, Marva (Indian), Arabian, Whole Tone, Blues, Pentatonic Minor, Hirajoshi.
+- Added "play `ONCE`" clip launch mode, settable by turning the Select Encoder left while holding the audition pad in Song view. Causes the clip to play once when triggered and then mute without changing the section.
+- Added `NOT FILL` note probability. Similar to the `FILL` probability but only plays when the `FILL` button is *not* pressed.
+- Added support for copy/paste of single rows.
+- Added support for "gentle paste" of notes which pastes notes without removing old ones.
+- Fixed numerous crash bugs around parameter automation when entering and leaving clip view.
+
+### Audio Clips
+- Changed default ModFX type is now DISABLED rather than FLANGER.
+
+### Kits
+
+- Drum randomization is no longer limited to only 10 sounds per folder.
+- Fixed several crashes related to drum randomization.
+
+### MIDI
+
+- Added `MIDI FOLLOW MODE` which causes MIDI data to be directed to the currently focused clip.
+- Added MIDI feedback, so external MIDI controllers can be made aware of the state of the Deluge synth engine. Configurable via the new global `MIDI > MIDI-FOLLOW > FEEDBACK` menu.
+- Added Loopy Pro and TouchOsc templates for MIDI feedback.
+- Added a `MIDI LOOPBACK` mode, accessible in the SONG menu, which directs MIDI data from internal MIDI clips back to the Deluge input.
+- Added support for learning Program Change methods to most global commands.
+- Added "MPE collapse" on MIDI clips which converts MPE X/Y/Z to Pitch/Modwheel/Aftertouch CCs for use of MPE controllers with non-MPE aware synths. Configurable via the clip menu.
+- Added a new global setting, `MIDI > SELECT KIT ROW`, which causes MIDI notes sent to kits to be sent to the individual row. Useful for CC control of individual kit rows.
+- Fixed reversal of upstream MIDI ports (port 1 was upstream port 2, port 2 was upstream port 1).
+- Fixed a number of MPE channel matching and learning bugs.
+- Fixed crash when using an external controller to control ModFX.
```

Comment:
Speaking of ModFX, should we mentioned that for Audio Clips we now allow Mod FX to be Disabled in Audio Clips, Song and Kit Affect Entire and are set to Disabled by default (they were set to Flanger by default previously and could not be disabled).